src/ServicePulse.Host/app/react/src/Footer.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+function Footer() {
+  return (
+
+    <footer className="footer ng-scope">
```

Comment:
This is just copied from the AngularJS index page. I didn't put any smarts into it